Done exercise(first version)

peopleAlreadyKnowWhoIAm opened the pull request:
Updates:

    - new file:   ua/lpnu/students/labs/first/Main.java
    - new file:   ua/lpnu/students/labs/first/hotel/Hotel.java
    - new file:   ua/lpnu/students/labs/first/hotel/HotelStatus.java

peopleAlreadyKnowWhoIAm committed:
Adding all files needed to the exercise(initial)

Updates:
    new file:   screen_of_the_exercise.png
    new file:   ua/lpnu/students/labs/first/Main.java
    new file:   ua/lpnu/students/labs/first/hotel/Hotel.java
    new file:   ua/lpnu/students/labs/first/hotel/HotelStatus.java

peopleAlreadyKnowWhoIAm committed:
Delete picture from this commit (Added to master-branch)
Updates:

	- deleted:    screen_of_the_exercise.png

orestv commented on ua/lpnu/students/labs/first/hotel/Hotel.java:
```diff
@@ -0,0 +1,55 @@
+    @Override
+    public String toString(){
+        return 
+            name + "\n\t" + 
+            visitorsMessage + ((numOfVisitorsPerYear == -1)? noneInfMessage: numOfVisitorsPerYear) +"\n\t"+
+            roomsMessage + ((numOfRooms == -1)? noneInfMessage: numOfRooms) +"\n\t"+
+            statusMessage + status.toString() +"\n\t"+
+            dinnerMessages[((hasDinner == true)?0:1)]+"\n\t"+
+            adressMessage + adress
```
Consider using `String.format` for this whole string.

peopleAlreadyKnowWhoIAm replied:
Did that in the last commit

peopleAlreadyKnowWhoIAm committed:
Edited acording to code review
	modified:   ua/lpnu/students/labs/first/hotel/Hotel.java
        Using String.format() instead of a long String adding

peopleAlreadyKnowWhoIAm committed:
Organisation fixes

Changes to be committed:
	renamed:    ua/lpnu/students/labs/first/Main.java -> ua/lpnu/students/labs/laba1/Main.java
	renamed:    ua/lpnu/students/labs/first/hotel/Hotel.java -> ua/lpnu/students/labs/laba1/hotel/Hotel.java
	renamed:    ua/lpnu/students/labs/first/hotel/HotelStatus.java -> ua/lpnu/students/labs/laba1/hotel/HotelStatus.java

peopleAlreadyKnowWhoIAm committed:
Merge pull request #2 from peopleAlreadyKnowWhoIAm/laba1

Organisation fixes
